libvips/foreign/tiff2vips.c

```diff
@@ -1880,6 +1940,11 @@ rtiff_stripwise_generate( VipsRegion *or,
 		y += hit.height;
 	}
 
+	/* Shut down the input file as soon as we can. 
+	 */
+	if( y >= or->im->Ysize )
```

Comment:
I think that `r-> top` should be subtracted from `y` here, `if( y - r->top >= or->im->Ysize )` (?)